File: project_templates/server_openapi3_starter/java/SecurityHandler.java
package {{#if project_info.package}}{{ project_info.package }}.{{/if}}securityHandlers;

import io.vertx.core.Handler;
import io.vertx.ext.web.RoutingContext;

public class {{ class_name }} implements Handler<RoutingContext> {

    public {{ class_name }}(){

    }

    @Override
    public void handle(RoutingContext routingContext) {
        // Handle your security
        routingContext.next();
    }

}

File: project_templates/server_openapi3_starter/kotlin/SecurityHandler.kt
package {{#if project_info.package}}{{ project_info.package }}.{{/if}}securityHandlers;

import io.vertx.core.Handler;
import io.vertx.ext.web.RoutingContext;

class {{ class_name }}: Handler<RoutingContext> {

    override fun handle(routingContext: RoutingContext) {
        // Handle your security
        routingContext.next()
    }

}